libImaging/Resample.c

```diff
@@ -184,7 +184,6 @@ precompute_coeffs(int inSize, int outSize, struct filter *filterp,
                 } else if (x == xmax - 1) {
                     xmax -= 1;
                 }
-                continue;
             }
             k[x] = w;
```

Comment:
> It does look like a calloc instead of malloc would solve all the initialization problems

This is true for this case. Of course, it would be better to initialize memory one extra time than get a bug. But I think it is unnecessary for now when we exactly know what everything is initialized. I also eliminated the last `continue` from this code.